llvm-spirv/test/transcoding/AtomicFAddEXT.ll

```diff
@@ -14,7 +14,7 @@
 ; RUN: llvm-dis %t.rev.bc -o - | FileCheck %s --check-prefixes=CHECK-LLVM-SPV
 
 target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-n8:16:32:64"
-target triple = "spir64-unknown-unknown-sycldevice"
+target triple = "spir64-unknown-unknown"
```

Comment:
> Since we anyway need to support object files generated by older versions of toolchain, there should be no rush with updating all those tests, right?

As I already mentioned [here](https://github.com/intel/llvm/pull/3929#issuecomment-896667850), the translator ignores this triple component, so it doesn't matter. 

@mlychkov, @AlexeySachkov, I'm going to revert the patch updating SPIR-V tests. I'll create a separate PR in the translator repository. BTW, if anyone wants to help with that it would be appreciated ;).

Reply:
> BTW, if anyone wants to help with that it would be appreciated ;)

@KornevNikita, could you please help with that?

Reply:
@AlexeySachkov Okay

Reply:
Thanks a lot!